src/Cli/dotnet/Commands/Tool/Install/ToolInstallLocalInstaller.cs

```diff
@@ -66,11 +66,17 @@ public IToolPackage Install(FilePath manifestFile, PackageId packageId)
 
         try
         {
+            //  NOTE: The manifest file may or may not be under a .config folder.  If it is, we will use
+            //  that directory as the root config directory.  This should be OK, as usually there won't be
```

Comment:
nit: Extra space in the comment

Reply:
Do you mean after the period?  I often put two spaces after the end of a sentence as that's what I learned when I was learning to type (on a mechanical typewriter!).